### ISSUE TITLE
Log filled with 'Can't start playing sample' messages

### ISSUE THREAD

**Issue by Loobinex** [Loobinex/keeperfx-unofficial#39]
In KeeperFX 0.4.6, both official and unofficial, whenever you play a level, the log file gets filled up with these messages:

```
Sync: play_sample_using_heap: Can't start playing sample 82
Sync: play_sample_using_heap: Can't start playing sample 172
```
It's mostly 172, sometimes 82 but generally no others. It seems there's something particularly wrong with these.

**Comment by Loobinex** [comment]
I've investigated a bit, and the game tries to play sound 172 when imps reinforce walls. There is no sound 172.
See line 826 of creature_instances.c

**Comment by Loobinex** [comment]
Closed with 1e0bc3192257d1f55b78de016750ae0f4874a2f1
